src/MainGrid.vala

```diff
@@ -34,7 +34,7 @@ public class Badger.MainGrid : Gtk.Grid {
         column_spacing = 12;
         orientation = Gtk.Orientation.VERTICAL;
 
-        var heading = new Granite.HeaderLabel (_ ("Reminders"));
+        var heading = new Granite.HeaderLabel (_("Reminders"));
```

Comment:
```suggestion
        var heading = new Granite.HeaderLabel (_ ("Reminders"));
```

So Badger follows the convention of having a space between a method/function name and the opening parenthesis. That applies even if the name is one character, so I'm gonna keep this space.

Reply:
Umm, I don't seem to have permission to commit to your branch. I'll just merge it and then change in master.

Reply:
Okay understand, I though this was an error.